source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -404,7 +404,7 @@ - (void)processImageConcurrently:(std::shared_ptr<Image> const &)imageElem
                       // if view is available, set image to it, and continue image processing
                       if(view) {
                           view.image = img;
-                          if(imgElem->GetImageSize() == ImageSize::Auto || imgElem->GetImageSize() == ImageSize::Stretch || imgElem->GetImageSize() == ImageSize::None){
+                          if(img && (imgElem->GetImageSize() == ImageSize::Auto || imgElem->GetImageSize() == ImageSize::Stretch || imgElem->GetImageSize() == ImageSize::None)){
                               CGFloat heightToWidthRatio = img.size.height / img.size.width;
```

Comment:
>img.size.width [](start = 77, length = 14)

Isn't it possible for `img` to be non-nil but have zero width? This code would still crash then.

Reply:
(Crash or whatever NSLayoutConstraint does when given NaN or Inf.)

---
In reply to: [188021155](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188021155) [](ancestors = 188021155)

Reply:
thanks

---
In reply to: [188022216](https://github.com/Microsoft/AdaptiveCards/pull/1444#discussion_r188022216) [](ancestors = 188022216,188021155)